notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>28 pixels wide by 28 pixels high -&gt; 28x28</p><p> </p><p>Anyways convention is height first then width</p><p> </p><p>I would drop "(a list of lists)".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Ack. This is intended for 100-level developers. It's intended to be more verbose and explanatory.</p>